src/reducers/auth/register/initialState.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
@Manorlds-Eaglespark, we resolved this to a single file located at `src/reducers`. Please consider removing it.

Reply:
removed. Thanks.